packages/currency/src/chains/tron/data/tron.ts

```diff
@@ -0,0 +1 @@
+export const chainId = 'mainnet';
```

Comment:
Found here: https://medium.com/tronnetwork/how-to-adapt-to-optimization-of-the-return-value-of-chainid-opcode-in-tvm-6160029b64a0
```suggestion
export const chainId = 728126428;
```

Reply:
ah, good find! 👍